src/audio/google_rtc_audio_processing.c

```diff
@@ -83,6 +83,35 @@ static int google_rtc_audio_processing_params(
 	return 0;
 }
 
+static int google_rtc_audio_processing_reconfigure(struct comp_dev *dev)
+{
+	struct google_rtc_audio_processing_comp_data *cd = comp_get_drvdata(dev);
+	uint8_t *config;
+	size_t size;
+	int ret;
+	size_t i;
+
+	comp_info(dev, "google_rtc_audio_processing_reconfigure()");
+
+	config = comp_get_data_blob(cd->tuning_handler, &size, NULL);
+	if (!config || !size) {
+		comp_err(dev, "google_rtc_audio_processing_reconfigure(): Tuning config not set");
```

Comment:
you emit an error but return 0

Reply:
Indeed.
I splitted in 2 case:
  One where we do not have config data and silently exit.
  One wherewe should have some and get no pointer.